skbio/stats/distance/_base.py

```diff
@@ -371,6 +371,12 @@ def plot(self, cmap=None, title=""):
 
         ax.set_xticklabels(self.ids, rotation=90, minor=False)
         ax.set_yticklabels(self.ids, minor=False)
+
+        # Ensure there is no white border around the heatmap by manually
+        # setting the limits
+        ax.set_ylim(0, len(self.ids))
```

Comment:
Moving these lines above `ax.invert_yaxis()` puts the vertical order back to what it is in master. Which orientation is preferred? I don't have strong feelings either way.

cc: @gregcaporaso @ebolyen